tests/src/testAssembler.cpp

```diff
@@ -31,8 +39,9 @@ auto SimpleAssemblersTest() {
   std::array<int, 2> elementsPerDirection = {2, 1};
   auto grid                               = std::make_shared<Grid>(bbox, elementsPerDirection);
 
-  for (int i = 0; i < 4; ++i) {
-    auto gridView = grid->leafGridView();
+  for (int ref = 0; ref < 4; ++ref) {
+    auto gridView  = grid->leafGridView();
+    auto totalDOFs = 2 * gridView.size(2);
```

Comment:
`totalDOFs`should be populated with `basis.dimension()` and not artificially through number of vertices